moped-editor/src/components/GridTable/GridTable.js

```diff
@@ -451,7 +465,12 @@ const GridTable = ({ title, query }) => {
                                           )}
                                         </RouterLink>
                                       )
-                                    ) : isAlphanumeric(column) ? (
+                                    ) : query.config.columns[column]?.link ? (
+                                      query.getFormattedValue(
+                                        column,
+                                        buildLinkData(row, column))
+                                    )
+                                    : isAlphanumeric(column) ? (
```

Comment:
check to see if the column should be turned into a link, and format the information based on the query